src/main/java/blackjack/domain/user/Money.java

```diff
@@ -0,0 +1,41 @@
+package blackjack.domain.user;
+
+import java.util.Objects;
+
+public class Money {
+
+    private static final String MIN_MONEY_ERROR = "[ERROR] 배팅 금액은 0원이 될 수 없습니다.";
+    private final double money;
```

Comment:
```suggestion
    private static final String MIN_MONEY_ERROR = "[ERROR] 배팅 금액은 0원이 될 수 없습니다.";

        private final double money;
```

클래스 변수와 인스턴스 변수 사이에 공백이 존재한다면 가독성이 상승합니다